lib/comgr/test/CMakeLists.txt

```diff
@@ -118,6 +119,13 @@ add_executable(get_data_isa_name_test get_data_isa_name_test.c)
 target_link_libraries(get_data_isa_name_test amd_comgr)
 add_test(NAME get_data_isa_name_test COMMAND get_data_isa_name_test)
 
+# Test : Compile HIP
+if (UNIX)
```

Comment:
If the HIP actions are unavailable on Windows I think it might be useful to only compile them in conditionally, similar to how device-libs is currently handled, or at least note this in the documentation for the language/action variants.

Reply:
support of HIP on windows is under development. The condition put on the test will be removed once HIP on windows is working.